package.json

```diff
@@ -87,10 +87,10 @@
   },
   "dependencies": {
     "@cocos/box2d": "1.0.1",
-    "@cocos/bullet": "1.0.7",
+    "@cocos/bullet": "1.0.8-exp.cct.4",
     "@cocos/cannon": "1.2.8",
     "@cocos/ccbuild": "^1.0.0-alpha.7",
     "@cocos/dragonbones-js": "^1.0.1",
-    "@cocos/physx": "1.0.7"
+    "@cocos/physx": "1.0.9-beta.cct.2"
```

Comment:
we've removed these npm dependencies, and put them into external repo, please take a look at this workflow docs
https://github.com/cocos/3d-tasks/blob/master/design-docs/scripting/engine-wasm-module.md

Reply:
sure